automation/run-integration-tests.sh

```diff
@@ -1,11 +1,16 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
 
 EXEC_PATH=$(dirname "$(realpath "$0")")
 PROJECT_PATH="$(dirname $EXEC_PATH)"
 : ${DOCKER_IMAGE:=nmstate/centos7-nmstate-dev}
 EXPORT_DIR="$PWD/exported-artifacts"
 CONT_EXPORT_DIR="/exported-artifacts"
-PYTHON27_PATH="/usr/lib/python2.7/site-packages"
+if [[ $DOCKER_IMAGE == *"fedora"* ]];then
+    PYTHON_SITE_PATH_CMD="rpm -E %{python3_sitelib}"
```

Comment:
There is a python command that can be run instad of querying rpm here, then it does not depend on the docker image name

Reply:
We still need the docker name to determine to run python3 or python2.
I failed to find a python command provides `single` site path and using rpm macro could better
aligned with nmstate installed by rpm.

Reply:
I guess we could default to python3 and fallback to python2 if it is not available.